src/manifest.json

```diff
@@ -66,6 +66,7 @@
         "js/shared/utils.js",
         "js/other-websites/fathom.js",
         "js/other-websites/email_detector.js",
+        "js/other-websites/inpage_menu.js",
```

Comment:
Hmm, I might be misunderstanding how this script work, but isn't this supposed to only be included within the iframe, rather than in every website?

Reply:
Ahhh - Good call. Removed in 4344dc8e29ad93376f4af2c58eebb7a81182ed08 and everything still seemed to work! 🎉